app/assets/stylesheets/dance.scss

```diff
@@ -0,0 +1,3 @@
+#dance_comment_list li {
```

Comment:
This is true! In general you want to use classes for sections of the page, and if you need to, use ids for records in a list:

```
<%= @cars.each do |car| %>
  <div id="car-<%= car.id %>"><%= car.make %></div>
<% end %>
```